### DISCUSSION TITLE
Action Summary Page on VS code

### DISCUSSION THREAD

**Discussion by SrinivasKashi** [community/community#60165] [Actions]
Anyway we can display the action summary page on vs code ?

**Comment by GhostOf0days** [comment]
Please look into the [VSCode Github Actions Extension](https://marketplace.visualstudio.com/items?itemName=GitHub.vscode-github-actions).

**Reply by SrinivasKashi** [reply]
It doesn't talk about the summary page on VS code

**Reply by GhostOf0days** [reply]
If there's a way to do it, it's through there. That is the only official GitHub Actions extension. It was released on March 28th of this year, so it probably will get new updates.

**Reply by SrinivasKashi** [reply]
Cool. Thanks

**Reply by GhostOf0days** [reply]
Hi. Could you please close this thread if the issue is fixed?
